src/Components/Artwork/Details/Details.tsx

```diff
@@ -355,12 +401,16 @@ export const Details: React.FC<React.PropsWithChildren<DetailsProps>> = ({
   }
 
   return (
-    <Box>
+    <Box height={CONTAINER_HEIGHT + "px"}>
```

Comment:
fwiw, shouldn't need to provide "px" (but also fine)

Reply:
Good to know! I thought it didn't work without it but I could be wrong.